setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name='OpenFisca-Country-Template',
-    version='3.0.2',
+    version='3.0.3',
```

Comment:
Changelog and setup versions do not match.

> I prefer the minor bump over the patch bump 🙂

Reply:
Done